cli/exchange/node.go

```diff
@@ -4,19 +4,21 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
+	"net/http"
+	"os"
+
 	"github.com/open-horizon/anax/cli/cliconfig"
 	"github.com/open-horizon/anax/cli/cliutils"
 	"github.com/open-horizon/anax/common"
 	"github.com/open-horizon/anax/compcheck"
 	"github.com/open-horizon/anax/exchange"
+	exch "github.com/open-horizon/anax/exchange"
```

Comment:
I have mistakenly done this. I had to import this in hzn.go file because there was a conflict with cli/exchange package in hzn.go file. Yes, I did a mistake here, I will remove it and I will use the existing one.